src/style/grouprow.ts

```diff
@@ -1,9 +1,8 @@
 import { css } from '@linaria/core';
-import { rowSelected } from './row';
 import { cell, cellFrozenLast } from './cell';
 
 const groupRow = css`
-  &:not(.${rowSelected}) {
+  &:not(aria-selected="true") {
```

Comment:
I don't think this works without the brackets.
```suggestion
  &:not([aria-selected="true"]) {
```